lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -0,0 +1,164 @@
+;(function (define) {
+'use strict';
+
+define(['backbone',
+        'underscore',
+        'gettext',
+        'js/views/fields',
+        'teams/js/models/team',
+        'text!teams/templates/edit-team.underscore'],
+       function (Backbone, _, gettext, FieldViews, TeamModel, edit_team_template) {
+           return Backbone.View.extend({
+
+               maxTeamNameLength: 255,
+
+               initialize: function(options) {
+                   this.courseId = options.teamParams.courseId;
+                   this.teamsUrl = options.teamParams.teamsUrl;
+                   this.topicId = options.teamParams.topicId;
+                   this.topicName = options.teamParams.topicName;
+                   this.languages = options.teamParams.languages;
+                   this.countries = options.teamParams.countries;
+
+                   this.eventAggregator = options.eventAggregator;
+                   _.bindAll(this, "cancelTeam", "createTeam");
+                   this.eventAggregator.bind("cancelTeam", this.cancelTeam);
+                   this.eventAggregator.bind("createTeam", this.createTeam);
+
+                   this.teamNameField = new FieldViews.TextFieldView({
+                       model: new TeamModel(),
+                       title: gettext("Team Name (Required) *"),
+                       valueAttribute: 'name',
+                       bindEvents: false,
+                       helpMessage: gettext("The name that will identify your team")
+                   });
+
+                   this.teamDescriptionField = new FieldViews.TextareaFieldView({
+                       model: new TeamModel(),
+                       title: gettext("Team Description (Required) *"),
+                       valueAttribute: 'description',
+                       editable: 'always',
+                       showMessages: false,
+                       bindEvents: false,
+                       descriptionMessage: gettext("A short description of the team to help other students understand the goals or directives the team is pursuing")
```

Comment:
Is there a length restriction on the description, like the one for the team name?